tests/test_classifier_collection.py

```diff
@@ -9,7 +9,7 @@
 
 from smqtk_descriptors.impls.descriptor_element.memory import DescriptorMemoryElement
 
-from tests.interfaces.test_classifier import DummyClassifier
+from tests.interfaces.test_classify_descriptor import DummyClassifier
 
 
 class TestClassifierCollection (unittest.TestCase):
```

Comment:
Similar to an above comment, if the classifier collection class is renamed to reflect its "ClassifyDescriptor" heritage, this should of course be renamed as well.